core/config/project_settings.cpp

```diff
@@ -41,6 +41,8 @@
 #include "core/os/keyboard.h"
 #include "core/os/os.h"
 #include "core/variant/variant_parser.h"
+#include "core/version.h"
+#include "modules/modules_enabled.gen.h"
```

Comment:
To follow the same style as #54912:
```suggestion
#include "core/version.h"

#include "modules/modules_enabled.gen.h" // For mono.
```